treeherder/settings/base.py

```diff
@@ -329,6 +331,35 @@
 # set this normally is your pulse username.
 PULSE_EXCHANGE_NAMESPACE = None
 
+# Specifies the Pulse exchanges Treeherder will ingest data from.  This list
+# will be updated as new applications come online that Treeherder supports.
+# Can be overridden in local.py to specify fewer or completely different
+# exchanges for testing purposes on local machines.
+PULSE_DATA_INGESTION_EXCHANGES = env.json(
+    "PULSE_DATA_INGESTION_EXCHANGES",
+    default=[
+        # {
+        #     "name": "exchange/taskcluster-treeherder/jobs",
+        #     "projects": [
+        #         'mozilla-central',
+        #         'mozilla-inbound'
+        #         # other repos TC can submit to
+        #     ]
+        # },
+        # {
+        #     "name": "exchange/treeherder-test/jobs",
+        #     "projects": [
+        #         'mozilla-inbound'
+        #     ]
+        # }
+        # ... other CI systems
+    ])
+
+# Used to specify the PulseGuardian account that will be used to create
+# ingestion queues for the exchanges specified in ``PULSE_DATA_INGESTION_EXCHANGES``.
+PULSE_DATA_INGESTION_CONFIG = env.url("PULSE_DATA_INGESTION_CONFIG", default="")
```

Comment:
Could you add an example URL to the comment?
Also I wonder if `None` makes more sense as a default?